recipe_scrapers/tineno.py

```diff
@@ -1,97 +1,93 @@
+import re
+from typing import Optional, Tuple, Union
+
 from ._abstract import AbstractScraper
-from ._utils import get_minutes, get_yields, normalize_string
+from ._utils import change_keys
+
+
+def fix_json_ld_property_keys(k):
+    if k == "ItemListElement":
+        return "itemListElement"
+    return k
 
 
 class TineNo(AbstractScraper):
+    def __init__(
+        self,
+        url,
+        proxies: Optional[str] = None,
+        timeout: Optional[Union[float, Tuple, None]] = None,
+        wild_mode: Optional[bool] = False,
+    ):
+        super().__init__(url, proxies, timeout, wild_mode)
```

Comment:
What do you think about modifying the method signature and superclass call here to insulate the scraper (somewhat) against future interface changes?
```suggestion
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
```

Reply:
Good suggestion :+1:, I will update this